libr/core/cmd.c

```diff
@@ -2453,6 +2456,25 @@ static int autocomplete_type(const char* strflag) {
 	return R_CORE_AUTOCMPLT_END;
 }
 
+static void cmd_autocomplete_help(RCore *core) {
+	r_core_cmd_help (core, help_msg_triple_exclamation);
+	// non-zero-cost survival without iterators 101
+	const char **help = malloc(R_CORE_AUTOCMPLT_END * sizeof(char *) * 3 + 1);
```

Comment:
missing space before (

Reply:
ah, OK, I'm still trying to get used to this code style.